DashWallet/Sources/UI/Explore Dash/Merchants & ATMs/List/Filters/PointOfUseListFiltersViewController.swift

```diff
@@ -188,13 +185,23 @@ class PointOfUseListFiltersViewController: UIViewController {
         configureHierarchy()
         configureDataSource()
         reloadDataSource()
+        
+        NotificationCenter.default.addObserver(self, 
+                                               selector: #selector(enteredBackround),
+                                               name: UIApplication.didEnterBackgroundNotification,
+                                               object: nil)
     }
 
     class func controller() -> PointOfUseListFiltersViewController {
         let storyboard = UIStoryboard(name: "ExploreDash", bundle: nil)
         return storyboard
             .instantiateViewController(withIdentifier: "PointOfUseListFiltersViewController") as! PointOfUseListFiltersViewController
     }
+    
+    @objc
+    func enteredBackround() {
+        dismiss(animated: false)
```

Comment:
Dismiss filter dialog if the user switches the app as they might be in the settings to change the location permission. The filters will be refreshed on re-open.

Dynamic view update might be better here, but more complicated.